tests/api-mongo.test.js

```diff
@@ -466,6 +466,30 @@ describe('MongoDB Words', () => {
         });
     });
 
+    it('should return a sorted list of igbo terms when using english by using single qoutes', (done) => {
```

Comment:
I would create another test that wraps an Igbo word in quotes and expect to get no results